zuul.d/jobs.yaml

```diff
@@ -367,20 +359,42 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/ansible.yang
 
+- job:
+    name: ansible-test-units-ansible-yang-python39
+    parent: ansible-test-units-base-python39
+    required-projects:
+      - name: github.com/ansible-collections/ansible.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/ansible.yang
+
 - job:
     name: ansible-test-network-integration-ansible-yang-iosxr-netconf-python36
     parent: ansible-test-network-integration-iosxr-python36
     vars:
       ansible_collections_repo: github.com/ansible-collections/ansible.yang
       ansible_test_integration_targets: ""
 
+- job:
+    name: ansible-test-network-integration-ansible-yang-iosxr-netconf-python38
+    parent: ansible-test-network-integration-iosxr-python38
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/ansible.yang
+      ansible_test_integration_targets: ""
+
 - job:
     name: ansible-test-network-integration-ansible-yang-junos-vsrx-netconf-python36
```

Comment:
@rohitthakur2590 All the network-ee integration jobs for the network collections are tested only with py39. If you want to test this collection with py36, it can tested with ansible-2.9 and not devel, because 'ansible-core' requires a different Python:  '>=3.8'`